tests/crm/conftest.py

```diff
@@ -142,6 +144,51 @@ def set_polling_interval(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
     time.sleep(wait_time)
 
 
+def get_intf_list(duthost, tbinfo, enum_frontend_asic_index):
+    """ Return the interface list which would influence fdb entry by mac learning """
+    asichost = duthost.asic_instance(enum_frontend_asic_index)
+    mg_facts = asichost.get_extended_minigraph_facts(tbinfo)
+    intf_connect_with_ptf = []
+    for intf, intf_desc in mg_facts["minigraph_neighbors"].items():
+        if "Server" in intf_desc['name']:
+            intf_connect_with_ptf.append(intf)
+    return intf_connect_with_ptf
+
+
+def check_interface_status(duthost, intf_list, expected_oper='up'):
+    """ Check interface status """
+    output = duthost.command("show interface description")
+    intf_status = parse_intf_status(output["stdout_lines"][2:])
+    for intf in intf_list:
+        if intf not in intf_status:
+            logging.info("Missing status for interface %s" % intf)
+            return False
+        if intf_status[intf]["oper"] != expected_oper:
+            logging.info("Oper status of interface {} is {}, expected {}".format(intf, intf_status[intf]["oper"],
+                                                                                 expected_oper))
+            return False
+    return True
+
+
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
How much time this would introduce?

Reply:
It depends on how many ports need to be shutdown. Usually, around 20 ports.